.gitattributes

```diff
@@ -0,0 +1 @@
+**/*.mustache linguist-vendored=true
```

Comment:
@MikeRalphson thanks for the PR. Do you think we should add the following as well to ignore the auto-generated files?
```
samples/* linguist-vendored=true
```